lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -51,6 +51,18 @@ def reload_file(path)
     load path
   end
 
+  def reload_diff_files
+    output = %x(git diff --name-only 2>&1)
```

Comment:
This is pretty close, but for portability and reliability, we should use `Open3.capture2e`, which does not rely on a shell when passed an array of arguments https://ruby-doc.org/stdlib-2.5.1/libdoc/open3/rdoc/Open3.html#method-c-capture2e

```ruby
output, _status = Open3.capture2e *%w(git diff --name-only)
```